src/commands/utility/calendar.js

```diff
@@ -0,0 +1,29 @@
+const { SlashCommandBuilder } = require('discord.js');
+const {
+  scheduleEventNotification,
+} = require('../../cron/schedule-google-calendar');
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('calendar')
+    .setDescription('Replies with Pong!'),
+  async execute() {
+    try {
+      const events2 = [
```

Comment:
This is a testing artifact, but I think is not intended for prod use, please check